src/web3/KeychainLock.ts

```diff
@@ -55,15 +51,13 @@ export async function decryptPrivateKey(encryptedPrivateKey: string, password: s
   }
 }
 
-export async function storePrivateKey(
-  privateKey: string,
-  account: KeychainAccount,
-  password: string
-) {
+async function storePrivateKey(privateKey: string, account: KeychainAccount, password: string) {
```

Comment:
Can we put these in an object instead?

Reply:
Yes probably but it's unrelated to the concern of this PR.